tests/rptest/services/action_injector.py

```diff
@@ -0,0 +1,183 @@
+import dataclasses
+import random
+import time
+from threading import Thread
+from typing import Optional, List
+
+from ducktape.cluster.cluster import ClusterNode
+from ducktape.utils.util import wait_until
+
+from rptest.clients.types import TopicSpec
+from rptest.services.admin import Admin
+from rptest.services.failure_injector import FailureSpec, FailureInjector
+from rptest.services.redpanda import RedpandaService
+
+
+@dataclasses.dataclass
+class ActionConfig:
+    cluster_start_lead_time_sec: float
+    min_time_between_actions_sec: float
+    max_time_between_actions_sec: float
+    max_affected_nodes: Optional[int] = None
+
+    def random_time_in_range(self) -> float:
```

Comment:
Let's rename `random_time_in_range` to `time_between_actions`, it gives more context about method and explains what it's for

Reply:
fixed